examples/cdk-examples-java/pom.xml.t.js

```diff
@@ -77,14 +77,35 @@ process.stdout.write(`
 
     <dependencies>
         <dependency>
-            <groupId>com.amazonaws</groupId>
+            <groupId>software.amazon.jsii</groupId>
```

Comment:
out of interest - how are we releasing jsii? I noticed from the repo there that it's org.jsii - did we want to keep that? or is this an amazon-branded product?

Reply:
It'll be published under `software.amazon.jsii`, and supposedly all places were updated accordingly (somehow this one seems to have slept though).